### DISCUSSION TITLE
Jquery - swipebox - problemas na visualização com os botões .close .prev . next

### DISCUSSION THREAD

**Discussion by Noturno411** [community/community#106778] [Programming Help]
### Body

Sorry for any wrong post types, tags etc.

Hello, I'm experiencing a problem that I can't resolve. I made a gallery in swipebox. I made an image file called icons.png with the button designs ( X < > ) and called it in my code. The buttons are repeating themselves (it's becoming X X X ) or they should be the arrows on the right side. Can anyone help me because these X's are repeating and none of the icons.png files are drawn correctly? Thank you for any help.

Here's an excerpt from my swipebox.css file (if you want the whole thing, I'll send it) and a screenshot of the screen it's printing.

#swipebox-action #swipebox-prev,
#swipebox-action #swipebox-next,
#swipebox-action #swipebox-close {
  background-image: url(../images/icons.png);
  background-repeat: no-repeat;
  border: none !important;
  text-decoration: none !important;
  cursor: pointer;
  position: absolute;
  width: 50px;
  height: 50px;
  top: 0;
}

#swipebox-action #swipebox-close {
  background-position: 15px 12px 0 auto;
  left: 40px;
}

#swipebox-action #swipebox-prev {
  background-position: 15px 25px 0 auto;
  right: 100px;
}

#swipebox-action #swipebox-next {
  background-position: -38px 13px 0 auto;
  right: 40px;
}

#swipebox-action #swipebox-prev.disabled,
#swipebox-action #swipebox-next.disabled {
  filter: progid:DXImageTransform.Microsoft.Alpha(Opacity=30);
  opacity: 0.3;
}

[SCREEN] - note that there are some dots too, it looks buggy. Strange because calling an image should show exactly the image. Could it be some configuration in my swipebox.css that would be repeating these X X X?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by queenofcorgis** [comment]
Hi @Noturno411 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.
